lib/deps/buffer-browser.js

```diff
@@ -0,0 +1,2 @@
+// hey guess what, we don't need this in the browser
+module.exports = {};
```

Comment:
This line is probably not necessary? (module.exports is already an object by default, I mean.)